src/table/__integ__/table.test.ts

```diff
@@ -38,10 +38,10 @@ test(
     const header = createWrapper().findContainer().findHeader().toSelector();
     await browser.url('#/light/table/full-page-variant?visualRefresh=true');
     const page = new BasePageObject(browser);
-    await expect(browser.execute(extractHeight, header)).resolves.toEqual({ height: '101px', marginBottom: '0px' });
+    await expect(browser.execute(extractHeight, header)).resolves.toEqual({ height: '91px', marginBottom: '0px' });
```

Comment:
Nitpick: I think what this test tries to say is that the sum of the height and the bottom margin should stay the same on scrolling, so that the content below the header doesn't jump. After these changes, this is still true 👍, but I think this test would better assert just that equality (sum of height and bottom margin is the same before and after) rather than kind-of-magic numbers. Otherwise a similar update in the future could easily break this.

Reply:
Updated! All that seems to matter is that the height is not smaller after scroll, otherwise it tries to recalculate indefinitely and jumps between the two sizes.